rc-events-service/src/main/scala/com/epam/random_coffee/events/model/DummyEvent.scala

```diff
@@ -0,0 +1,18 @@
+package com.epam.random_coffee.events.model
+
+import java.time.Instant
+
+final case class EventId(value: String) extends AnyVal
+
+final case class DummyEvent(id: EventId, name: String)
+
+final case class Author(id: String) extends AnyVal
```

Comment:
Since it's an AnyVal, I'm assuming you are not expecting this class to have more fields in the future. Why don't call it a `UserId` and place to a separate file?